torch/_C/__init__.pyi.in

```diff
@@ -746,8 +746,15 @@ def __set_forward_AD_enabled(enabled: _bool) -> None: ...
 def __is_forward_AD_enabled() -> _bool: ...
 def _register_default_hooks(pack_hook: Callable, unpack_hook: Callable) -> None: ...
 def _reset_default_hooks() -> None: ...
-def _enter_python_mode(cls: Type) -> None: ...
-def _exit_python_mode() -> None: ...
+
+# Defined in torch/overrides.py
+class TorchFunctionMode(object):
+    ...
+
+def _set_torch_function_mode(cls: Optional[Union[type, TorchFunctionMode]]) -> None: ...
+def _get_torch_function_mode() -> Type: ...
```

Comment:
Also probably should be `Optional[Union[type, TorchFunctionMode]`? (I'm not sure about the Optional part, what happens if there is no mode?)

Reply:
it returns None